### ISSUE TITLE
getStreamDrops is always returning 0

### ISSUE THREAD

**Issue by Kohbrax** [xjsframework/xjs#133]
Im using XSplit 2.9.1701.1616 with the latest xjs api version.
XSplit is showing '300' dropped frames, but the api call is always returning '0'

Example Code:

```
xjs.StreamInfo.getActiveStreamChannels()
	.then((streamInfo) => {
		let drops = [ ];
		streamInfo.forEach((stream) => {
			drops.push(stream.getStreamDrops());
		});

		Promise.all(drops)
			.then((data) => {
				let totalDrops = 0;

				data.forEach((d) => {
					totalDrops += d;
				});
				
				console.log(totalDrops);
			});
})
```

even the value from `iApp.get('streamdrops:' + this._name).then(val => {` seems to be always zero.
But the current frames count is correkt.

**Comment by virn** [comment]
The API call `getStreamDrops` returns the value of the current dropped frames at the exact moment the call is made. Which is why you keep on getting 0 when you call it when there are no frame drops currently happening on your stream.

You may notice that while the frame drops are happening, this value is reflected on XBC and at some point would stay on a specific value if there are no drops happening anymore. This value is the **GOP Drop**, and is what XBC shows cumulatively. The **Dropped** status on XBC's title bar is used for both statistics.

We've requested changes for us to be able to get GOP drops as well and is filed on Internal XSplit item XBC-6258.

**Comment by Kohbrax** [comment]
Hello virn,

thanks for your investigation. So the current solution is, wo create an api call every millisecond to know if the stream drops frames?
My Feeling is, that this api-call is really unnessecary.

GOP Drops seems to be perfect - even more perfect would be an event if framedropps occour.

**Comment by virn** [comment]
For now, this is the only solution if you want to get the current stream drops when it happen. Though the API call should not be every millisecond since XBC emmits this event at about every 1 second.

The internal item to get GOP drops has been finished and we've added `getGOPDrop` on StreamInfo class, this will most likely be included on the next release of the framework **2.3.0**.
